### ISSUE TITLE
Remove some components of https://bchmempool.space/about

### ISSUE THREAD

**Issue by Rucknium** [Rucknium/bchmempool#8]

**Comment by HardenedSteel** [comment]
What components will be changed? Do we need remove sponsors?

**Comment by Rucknium** [comment]
@HardenedSteel Thank you for taking an interest in this project! Yes, I think it makes sense to remove the sponsors since they are not sponsoring `bchmempool`.

Overall, I am not sure about the etiquette of editing the page. I think sponsors can be removed, but maybe say that sponsors of the BTC mempool version can be viewed at this link: https://mempool.space/about .

As far as the programmers, I think we should keep the list of Project Contributors, Project Members, and Project Maintainers, but identify them as contributing to the BTC mempool version.

What do you think?

**Comment by HardenedSteel** [comment]
@Rucknium I agree about removing sponsors but keeping both BTC and BCH project contributors versions can be confusing and long list, instead maybe we can link https://mempool.space/about

**Comment by Rucknium** [comment]
@HardenedSteel Ok that sounds fine. Thanks for your help.

Within a few days we are going to have a new logo for the website by the way.

**Comment by HardenedSteel** [comment]
Still sponsor button should be fixed with #26
